rsyslog/Sanity/capabilities-drop/main.fmf

```diff
@@ -0,0 +1,21 @@
+summary: tests dropping of capabilities
+contact: Dalibor Pospíšil <dapospis@redhat.com>
+test: ./runtest.sh
+require+:
+  - /usr/bin/pscap
+  - library(ControlFlow/Cleanup)
+  - library(distribution/testUser)
+duration: 5m
+enabled: true
+adjust:
+  - enabled: false
+    when: distro < rhel-8.6
```

Comment:
We are targeting 9.2, could you change that?

Reply:
sure, that's just c&p error